services/account/README.md

```diff
@@ -209,6 +198,7 @@ Install NPM Dependencies:
 
 ```sh
   npm install
+  npm install -g @nestjs/cli
```

Comment:
Not sure about this one, but my system could not find the `nest` command.

Reply:
`account-service` uses `nest` in a bunch of the `package.json` scripts. I probably lost my `nest` when switching from asdf to nvm. We could also add `@nestjs/cli` to the dev-dependencies.

Reply:
> add @nestjs/cli to the dev-dependencies

At least on that commit it shows up as a full dependency: https://github.com/ProjectLibertyLabs/gateway/blob/859ab33ab96e359075085e3a4908d3ff8e8446a2/services/account/package.json#L56